package.json

```diff
@@ -3,7 +3,8 @@
   "scripts": {
     "start": "functions-framework --target=xplorersbot",
     "test": "task clean && jest --verbose --coverage tests",
-    "start-local-server": "task clean && tsc && cd out && npx functions-framework --target=xplorersbot --signature-type=http"
+    "start-xplorersbot-server": "task clean && tsc && cd out && npx functions-framework --target=xplorersbot --signature-type=http",
```

Comment:
it is a bit strange that package-lock.json changed without changing any dependencies. not a blocker though.